index.jsx

```diff
@@ -110,18 +115,18 @@ static _componentMap = new Map()
     let matchedSize = ''
     let matchedWidth = smallestSize.width
 
+    // use Array#some() here because #forEach() has no early exit
```

Comment:
Yeah, it took me a second to realise why that choice was made, and I figured it was worth spelling out. :)